desktop-environment.el

```diff
@@ -426,8 +445,8 @@ the screen."
   :global t
   :require 'desktop-environment
   :lighter " DE"
-  (when desktop-environment-mode
-    (desktop-environment-exwm-set-global-keybindings)))
+  (when desktop-environment-update-exwm-global-keys
```

Comment:
I think I would remove the `when` here to only keep the body and let `desktop-environment-exwm-set-global-keybindings` take care of `nil` itself.